src/components/EntitySettings/index.vue

```diff
@@ -60,6 +60,7 @@
               <button
                 class="btn btn-primary btn-rounded"
                 :disabled="inviteForm.userUuid === null || inviteStatus.isPending()"
+                data-cy="invite"
```

Comment:
Just wondering: `data-cy` (here and in other occurences) is cypress related?

Reply:
It is recommended by Cypress to select elements by data attributes instead of, e.g., CSS classes https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements